### PR TITLE
Make units remain after exit

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: units/systemd-network-generator.service.in [hunk 1]
```diff
@@ -14,6 +14,7 @@ Before=network-pre.target
 
 [Service]
 Type=oneshot
+RemainAfterExit=true
```
Comment (yuwata) [inline units/systemd-network-generator.service.in]: I prefer 'yes' rather than 'true'.

File: units/modprobe@.service [hunk 2]
```diff
@@ -16,4 +16,5 @@ ConditionCapability=CAP_SYS_MODULE
 
 [Service]
 Type=oneshot
+RemainAfterExit=yes
```
Comment (yuwata) [inline units/modprobe@.service]: Have you read the discussion in #14819?. I am not sure this is safe or not...

Comment (keszybz) [reply units/modprobe@.service]: Oh, I forgot about about that PR. I'll drop my patch then. Let's continue the discussion there.

**Comment by keszybz** [comment]
Updated to drop the conflicting commit and s/true/yes/.

**Comment by yuwata** [comment]
LGTM.

**Pull request merged**